packages/browser-wallet-api/package.json

```diff
@@ -21,6 +20,9 @@
         "jest": "^29.7.0",
         "ts-jest": "^29.1.1"
     },
+    "peerDependencies": {
+        "@concordium/web-sdk": "*"
+    },
```

Comment:
I'm actually not 100% certain that this is enough to ensure consistency between the versions used in wallet-api/wallet..

I did 
```
yarn dedupe @concordium/web-sdk
```

which aligned the version across all packages.. As the browser wallet depends on this in `inject/index.ts` I would hope the browser wallet version is used, but I might be wrong...

Reply:
Technically, it should work. 

But in the future, this may create additional complications with versions, somewhere there will be a `workspace^` and somewhere a version that can be checked only through `package.lock`.
In addition, there will be a risk that one version is installed locally and used during development, but during the build of the release package, updated one will be downloaded. Which adds complexity to package management.

Reply:
Okay.. Do you have a better suggestion, or should we just go with this? One alternative approach I also had in mind was to refactor the workspace into

```
examples/
  ...
packages/
  browser-wallet/
    src/
      ...
      message-hub
      wallet-api
  browser-wallet-api-helpers/
    ...
```

This eliminates the need to define packages individually for wallet and wallet-api, and there is no real need for them to be defined as separate packages.

The reason I went with the current approach for now was simplicity, but I think the one outlined above would be safer

Reply:
For context, structure proposed above was merged in with #490